scalyr_agent/builtin_monitors/postgres_monitor.py

```diff
@@ -401,3 +413,4 @@ def timestamp_ms(dt):
                             self._logger.emit_value(self._db._database_stats[table][key][0], dbstats[key], extra)
                         else:
                             self._logger.emit_value(self._db._database_stats[table][key][0], timestamp_ms(dbstats[key]))
+        self._db.close()
```

Comment:
Add in a comment explaining why you close this.  We don't want the bug to sneak back in .

Reply:
Done